src/main/java/io/conduit/SpecifierService.java

```diff
@@ -0,0 +1,101 @@
+            "wrapper.connector.class",
+            Specifier.Parameter.newBuilder()
+                    .setDescription(
+                            "The class of the requested connector. "
+                            + "It needs to be found on the classpath, i.e. in a JAR in the `libs` directory. ")
+                    .setDefault("none")
+                    .setRequired(true)
+                    .build()
+    );
+
+    private static final Map<String, Specifier.Parameter> DESTINATION_PARAMS = Map.of(
+            "wrapper.connector.class",
+            Specifier.Parameter.newBuilder()
+                    .setDescription(
+                            "The class of the requested connector. "
+                            + "It needs to be found on the classpath, i.e. in a JAR in the `libs` directory. ")
+                    .setDefault("none")
+                    .setRequired(true)
+                    .build(),
+
+            "wrapper.schema",
+            Specifier.Parameter.newBuilder()
+                    .setDescription("The schema of the records which will be written to a destination connector.")
+                    .setDefault("none")
+                    .setRequired(false)
+                    .build(),
+
+            "wrapper.schema.autogenerate.enabled",
+            Specifier.Parameter.newBuilder()
+                    .setDescription(
+                            "Automatically generate schemas (destination connector). "
+                            + "Cannot be `true` if a schema is set.")
+                    .setDefault("none")
+                    .setRequired(false)
+                    .build(),
+
+            "wrapper.schema.autogenerate.name",
+            Specifier.Parameter.newBuilder()
+                    .setDescription("Name of automatically generated schema. "
+                            + "Required if schema auto-generation is turned on.")
+                    .setDefault("none")
+                    .setRequired(false)
+                    .build(),
+
+            "wrapper.schema.autogenerate.overrides",
+            Specifier.Parameter.newBuilder()
+                    .setDescription("A (partial) schema which overrides types in the auto-generated schema.")
+                    .setDefault("none")
+                    .setRequired(false)
+                    .build()
+    );
+
+    @Override
+    public void specify(Request request, StreamObserver<Response> responseObserver) {
+        responseObserver.onNext(
+                Response.newBuilder()
+                        .setName("conduit-kafka-connect-wrapper")
+                        .setSummary("Kafka Connect wrapper")
+                        .setDescription(
+                                "Conduit's Kafka connector wrapper is to make it possible "
+                                + "to use existing Kafka connectors with Conduit.")
+                        .setVersion("0.2.0")
```

Comment:
[Conduit expects](https://github.com/ConduitIO/conduit-connector-protocol/blob/4b89a6c94401924eb11a2554298be0e5ab3b9b88/proto/connector/v1/connector.proto#L199-L201) a `v` prefixed semantic version here.
```suggestion
                        .setVersion("v0.2.0")
```